includes/class-instawp-rest-api.php

```diff
@@ -517,36 +535,43 @@ public function config( $request ) {
 	 *
 	 * @param WP_REST_Request $request
 	 *
-	 * @return void
+	 * @return WP_Error|bool
 	 */
-	function validate_api_request( WP_REST_Request $request ) {
+	private function validate_api_request( WP_REST_Request $request ) {
+
+		if ( ! defined( 'INSTAWP_ALLOW_MANAGE' ) || ( defined( 'INSTAWP_ALLOW_MANAGE' ) && true !== INSTAWP_ALLOW_MANAGE ) ) {
```

Comment:
@iamsayan 

This constant check is only for remote management, not for all REST API calls. There are 2 ways to check this - 

1. Check it inside where it is required. 
2. Pass an extra parameter to this method `validate_api_request ()` like `$check_management` and make it by default `false`. And inside the `validate_api_request()` method, if the `$check_management` is true only then do this check. So, pass this new param `true` when the method is calling inside from Remote Management.

My Preference is 2nd one.